### ISSUE TITLE
v3.0.6-b1 bug

### ISSUE THREAD

**Issue by thenoid** [gcobb321/icloud3_v3#344]
snagged the beta zip to fix the auth issue, seeing these errors in the logs

```
homeassistant  | 2024-10-26 10:05:29.397 ERROR (SyncWorker_8) [custom_components.icloud3] 'mobapp'
homeassistant  | Traceback (most recent call last):
homeassistant  |   File "/config/custom_components/icloud3/device.py", line 812, in _restore_mobapp_items_from_restore_state_file
homeassistant  |     mobapp_items = Gb.restore_state_devices[self.devicename]['mobapp']
homeassistant  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
homeassistant  | KeyError: 'mobapp'
homeassistant  | 2024-10-26 10:05:29.418 ERROR (SyncWorker_8) [custom_components.icloud3] 'mobapp'
homeassistant  | Traceback (most recent call last):
homeassistant  |   File "/config/custom_components/icloud3/device.py", line 812, in _restore_mobapp_items_from_restore_state_file
homeassistant  |     mobapp_items = Gb.restore_state_devices[self.devicename]['mobapp']
homeassistant  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
homeassistant  | KeyError: 'mobapp'
homeassistant  | 2024-10-26 10:05:29.463 ERROR (SyncWorker_8) [custom_components.icloud3] 'mobapp'
homeassistant  | Traceback (most recent call last):
homeassistant  |   File "/config/custom_components/icloud3/device.py", line 812, in _restore_mobapp_items_from_restore_state_file
homeassistant  |     mobapp_items = Gb.restore_state_devices[self.devicename]['mobapp']
homeassistant  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
homeassistant  | KeyError: 'mobapp'
homeassistant  | 2024-10-26 10:05:29.513 ERROR (SyncWorker_8) [custom_components.icloud3] 'mobapp'
homeassistant  | Traceback (most recent call last):
homeassistant  |   File "/config/custom_components/icloud3/device.py", line 812, in _restore_mobapp_items_from_restore_state_file
homeassistant  |     mobapp_items = Gb.restore_state_devices[self.devicename]['mobapp']
homeassistant  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
homeassistant  | KeyError: 'mobapp'
homeassistant  | 2024-10-26 10:05:29.533 ERROR (SyncWorker_8) [custom_components.icloud3] 'mobapp'
homeassistant  | Traceback (most recent call last):
homeassistant  |   File "/config/custom_components/icloud3/device.py", line 812, in _restore_mobapp_items_from_restore_state_file
homeassistant  |     mobapp_items = Gb.restore_state_devices[self.devicename]['mobapp']
homeassistant  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
homeassistant  | KeyError: 'mobapp'
homeassistant  | 2024-10-26 10:05:29.563 ERROR (SyncWorker_8) [custom_components.icloud3] 'mobapp'
homeassistant  | Traceback (most recent call last):
homeassistant  |   File "/config/custom_components/icloud3/device.py", line 812, in _restore_mobapp_items_from_restore_state_file
homeassistant  |     mobapp_items = Gb.restore_state_devices[self.devicename]['mobapp']
homeassistant  |                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
homeassistant  | KeyError: 'mobapp'
```

**Comment by gcobb321** [comment]
Closed with the v3.1.1 Release. Create a new issue if you continue to experience this problem.

**Comment by thenoid** [comment]
Thank boss.
